lib/hasher/src/ADLER32Strategy.cpp

```diff
@@ -0,0 +1,79 @@
+#include "ADLER32Strategy.hpp"
+#include "checksum.hpp"
+
+#include <string>
```

Comment:
This header can be removed because it is already included by `ADLER32Strategy.hpp`.

This should only apply to hpp/cpp pairs.

Reply:
done